packages/@adobe/spectrum-css-temp/components/textfield/skin.css

```diff
@@ -164,3 +164,32 @@ governing permissions and limitations under the License.
     fill: var(--spectrum-alert-error-icon-color);
   }
 }
+@media (forced-colors: active) {
+  .spectrum-Textfield-input {
+    --spectrum-textfield-text-color-disabled: GrayText;
+    --spectrum-textfield-border-color-disabled: GrayText;
+    --spectrum-textfield-quiet-border-color-disabled: GrayText;
+    --spectrum-textfield-quiet-border-color-key-focus: Highlight;
+    --spectrum-textfield-border-color-key-focus: Highlight;
+    --spectrum-textfield-placeholder-text-color: GrayText;
+    --spectrum-textfield-placeholder-text-color-hover: GrayText;
+    &.is-focused, &:focus {
+      outline: 2px solid Highlight;
+    }  
+    &.focus-ring, &:focus-ring {
+      outline: 2px solid Highlight;
+    }  
+    .spectrum-Textfield--quiet & {
+      &.focus-ring, &:focus-ring {
+        forced-color-adjust: none;
+        outline: 0;
+        box-shadow: 0 1px 0 0 var(--spectrum-textfield-quiet-border-color-key-focus);
+        .spectrum-Textfield-input {
+          border-color: var(--spectrum-textfield-quiet-border-color-key-focus);
+        }
+      }
+    }
+   
+
+  }
```

Comment:
In Firefox, the disabled state looks wrong. It has a white background. Seems ok in edge/chrome though.

Also, the invalid/valid icons in Firefox are still red/green rather than white.

Reply:
Firefox doesn't support forced-color-adjust:none at the moment. To be honest I don't think we have a hope of supporting FF in high contrast until they do unless I simplify the spectrum defined high contrast behaviour. 

I'm ok with either direction but as far as I know the high contrast usage with FF is not very high.

Reply:
@devongovett are we looking to support FF in WHCM?

Reply:
I'm happy to follow your lead there. If it's easy to support, then great, but if it's not possible or a lot of additional work for not much benefit than it's fine with me if we don't.